components/connector/internal/revocation/loader.go

```diff
@@ -0,0 +1,109 @@
+package revocation
+
+import (
+	"context"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+type Loader interface {
+	Run(ctx context.Context)
+}
+
+type revocationListLoader struct {
+	revocationListCache Cache
+	manager             Manager
+	configMapName       string
+	reconnectInternal   time.Duration
+}
+
+func NewRevocationListLoader(revocationListCache Cache,
+	manager Manager,
+	configMapName string,
+	reconnectInternal time.Duration,
+) Loader {
+	return &revocationListLoader{
+		revocationListCache: revocationListCache,
+		manager:             manager,
+		configMapName:       configMapName,
+		reconnectInternal:   reconnectInternal,
+	}
+}
+
+func (rl *revocationListLoader) Run(ctx context.Context) {
+	rl.startKubeWatch(ctx)
+}
+
+func (rl *revocationListLoader) startKubeWatch(ctx context.Context) {
+	for {
+		select {
+		case <-ctx.Done():
+			log.Println("context cancelled, stop revocation config map watcher")
```

Comment:
Why not get logger down to the revocationListLoader and use it. Valid for the whole file.

Reply:
I don't see any logger configured at all, so I decided to leave it as it is. When there is a proper logger implementation this can be done better

Reply:
Probably we can use logrus just to be consistent in the future, but still... Since there isn't any logging infrastructure I understand that we cannot do much at all. It is ok